core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -139,22 +104,12 @@ public void run() {
       incSignerLoadCount(metricsSystem, signers.getAvailableIdentifiersCount());
```

Comment:
We should really only be loading the signers applicable for the mode we are running in. Later ticket though probably?

Reply:
Can we make this a later ticket? Its an optimisation rather than a requirement I hope/think.